src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -85,6 +143,30 @@ public void setPerson(Person target, Person editedPerson) {
         persons.setPerson(target, editedPerson);
     }
 
+    /**
+     * Replaces the given person {@code target} in the list with {@code editedProject}.
```

Comment:
should be given project instead of given person?
```suggestion
     * Replaces the given project {@code target} in the list with {@code editedProject}.
```

Reply:
Change made